site/content/docs/getting-started/installation.md

```diff
@@ -1 +1,39 @@
-### TODO
\ No newline at end of file
+# Installing the deployment tool<a name="deployment-tool-setup"></a>
+
+
+****  
+
+The following instructions show you how to install, update, and uninstall the deployment tool\.
+
+**To install the deployment tool**
+
+1. Open a command prompt or terminal.
+
+1. Install the tool: `dotnet tool install --global aws.deploy.tools`
+
+1. Verify the installation by checking the version: `dotnet-aws --version`

```

Comment:
In run-tool.md, I've expressed both forms.
As far as I'm concerned, either is fine.
If you all have a preference, please let me know so that I can make adjustments.

Reply:
my personal preference is to use `dotnet aws` by default. Not sure what the team thinks about this.